tests/common.py

```
@@ -39,6 +42,26 @@
 StrPath: TypeAlias = str | os.PathLike
 
 
+class QuietHTTPRequestHandler(SimpleHTTPRequestHandler):
+    def log_message(self, format, *args):
```

Ruff (0.15.10)

[error] Function argument `format` is shadowing a Python builtin

(A002)